src/DependencyInjection/SonataTimelineExtension.php

```diff
@@ -178,4 +192,61 @@ public function registerDoctrineMapping(array $config)
             'orphanRemoval' => false,
         ]);
     }
+
+    private function registerSonataDoctrineMapping(array $config): void
+    {
+        foreach ($config['class'] as $type => $class) {
+            if (!class_exists($class)) {
+                return;
+            }
+        }
+
+        $collector = DoctrineCollector::getInstance();
+
+        $timelineActionOptions = OptionsBuilder::createManyToOne('action', $config['class']['action'])
```

Comment:
Doing this on a stable branch is very risky. The library could be loaded, but the bundle must no be loaded.

Reply:
There is a check to see if the bundle is loaded: 

```
        if (isset($bundles['SonataDoctrineBundle'])) {
            $this->registerSonataDoctrineMapping($config);
        } else {
            // NEXT MAJOR: Throw error when not registering SonataDoctrineBundle
            @trigger_error(
                'Using SonataEasyExtendsBundle is deprecated since sonata-project/timeline-bundle 3.x. Please register SonataDoctrineBundle as a bundle instead.',
                E_USER_DEPRECATED
            );

            $this->registerDoctrineMapping($config);
        }
```

Reply:
The problem is, that we remove a required dependency that could potential break your application if you do not use the new one. 
On the other hand, the user should have required the `EasyExtends` in their `composer.json`, so it should be no problem.

Reply:
Yes, thats the point, we do not force to migrate to the new way of registering doctrine mapping but you should at least have the easy extends as a require dependency. We use this rule a lot of times, because if you are registering the easy extends bundle you should declare explicitly this dependency.

It is similar to the removal of CoreBundle, we do not have it anymore on require but you can continue using it.